.github/workflows/ci_tests_dev.yaml

```
@@ -98,8 +98,13 @@ jobs:
       # Pull baseline image data from dvc remote (DAGsHub)
       - name: Pull baseline image data from dvc remote
         run: |
+          ORIGINAL_BRANCH=$(git branch --show-current)
+          # Pull down GMT 6.4 baseline images from
+          # https://github.com/GenericMappingTools/pygmt/pull/1791
```

Also need to update the comment
```suggestion
          # Pull down GMT 6.4 baseline images from the gmt-6.4 branch
```